docs/admin/installation/hardware.rst

```diff
@@ -449,8 +450,33 @@ NUC before it can be used. We recommend:
 -  1x memory kit of compatible 2x8GB sticks
    -  You can put one 8GB memory stick in each of the servers.
 
-.. _nucc12_recommendation:
-.. _intel-12th-gen-nuc:
+.. _nuc13_recommendation:
+
+ASUS 13th-gen NUC
```

Comment:
Can we avoid calling it "ASUS ..." after we removed "Intel" from the other headers?

Reply:
Done.